controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -151,14 +155,33 @@ func (r *StorageClassClaimReconciler) Reconcile(ctx context.Context, request rec
 }
 
 func (r *StorageClassClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	enqueueStorageConsumerRequest := handler.EnqueueRequestsFromMapFunc(
```

Comment:
Name here is very generic, can we change that to `enqueueStorageConsumerRequestFromAnnotation`